frontend/src/core/navigation/index.js

```diff
@@ -0,0 +1,14 @@
+/* @flow */
+
+// import * as actions from './actions';
+import * as constants from './constants';
+import reducer from './reducer';
+
+export type { Action, State } from './reducer';
```

Comment:
What is the purpose of exposing `Action` and `State` as public interface of the module?

Reply:
Good catch, that's some leftover from when I was struggling with Flow.